deps.list

```diff
@@ -26,5 +26,5 @@ PROTOC_GEN_VALIDATE_VERSION=v0.9.1
 RUST_IMAGE_VERSION=1.67.0-alpine3.17
 SCALA_SBT_IMAGE_VERSION=graalvm-ce-21.3.0-java17_1.8.0_3.2.1
 SWIFT_IMAGE_VERSION=5.7-focal
-UPX_VERSION=4.0.1
+UPX_VERSION=v4.0.2
```

Comment:
Looks like https://github.com/rvolosatovs/docker-protobuf/blob/main/.github/renovate.json#L182 needs to be updated to trim the `v` prefix?

Reply:
Yep, I'll fix that with an `extractVersionTemplate`. 

This version of UPX also fixes several arm64 issues, so I want to test this to see if we can remove some of the arch conditions in the build.